ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -158,7 +160,7 @@ public ReactViewGroup createViewInstance(ThemedReactContext context) {
 
   @Override
   public Map<String, Integer> getCommandsMap() {
-    return MapBuilder.of("hotspotUpdate", CMD_HOTSPOT_UPDATE, "setPressed", CMD_SET_PRESSED);
+    return MapBuilder.of("hotspotUpdate", CMD_HOTSPOT_UPDATE, "setPressed", CMD_SET_PRESSED, "playClickSound", CMD_PLAY_CLICK_SOUND);
```

Comment:
Nit: this line is getting pretty long can you split it with one key/value pair per line?